src/cljc/rems/application_util.cljc

```diff
@@ -11,7 +11,7 @@
   (contains? (:application/permissions application)
              :application.command/save-draft))
 
-(def ^:private name-attribute-priority [:name :commonName :displayName :eppn])
+(def ^:private name-attribute-priority [:name :commonName :displayName :userid :eppn])
```

Comment:
Could this be removed, if `:name` is anyways required?

Reply:
done